templates/user/terminate_membership.html.twig

```diff
@@ -0,0 +1,125 @@
+{% extends 'base.html.twig' %}
+
+{% block page_title 'Désadhésion' %}
+
+{% block javascripts %}
+    {% import 'javascript.js.twig' as js %}
+
+    <script type="text/javascript" src={{ asset('bundles/sonataadmin/vendor/jquery/dist/jquery.min.js') }}></script>
+    <script type="text/javascript" src={{ asset('bundles/sonataadmin/vendor/jqueryui/ui/jquery-ui.js') }}></script>
+    <script type="text/javascript">
+        $(document).ready(function() {
+            var anotherReasons = $('#unregistration_user_reasons_5');
+            var toggleComment = function() {
+                if (anotherReasons.is(':checked')) {
+                    $('#unregistration_user_another_reason_comment').show();
+                } else {
+                    $('#unregistration_user_another_reason_comment').hide();
+                }
+            };
+
+            // Show/hide comment field
+            anotherReasons.change(function() {
+                toggleComment();
+            });
+
+            toggleComment();
+        });
+    </script>
+{% endblock %}
+
+{% block content %}
+
+<div>
+    {% if not unregistered %}
+        <div id="is_adherent" class="settings__unsub">
+            <header class="filtered-image progressive-background"
+                    data-sd="{{ static_asset('em-marcheurs-bg.jpg', { w: 300, q: 20 }) }}"
+                    data-hd="{{ static_asset('em-marcheurs-bg.jpg', { q: 70 }) }}">
+                <div class="l__wrapper--narrow">
+                    <h1 class="text--center text--larger">
+                        Je souhaite supprimer mon compte <em>En Marche</em><br/>
+                        Nous sommes tristes de vous voir partir 😢
+                    </h1>
+                </div>
+            </header>
+            <section class="l__wrapper--600 settings__unsub__intro">
+                <p>
+                    Merci de nous aider à comprendre pourquoi vous souhaitez quitter En Marche en choisissant
```

Comment:
En marche -> `La République En Marche`?

Reply:
I asked @guillaumedenhez for this question, we can keep "En Marche" for the moment :+1: